.github/workflows/lint-chart.yaml

```diff
@@ -17,6 +17,10 @@ on:
     paths:
       - "charts/**"
 
+permissions:
+  contents: read # Required by actions/checkout@v4 to fetch the repository contents.
+  statuses: write # Necessary to update the status of workflow runs.      
```

Comment:
```suggestion  
```
permission `statuses: write` is unnecessary

Reply:
@zhzhuang-zju as said by @liangyuanpeng in #5082 I have opened another PR #5126 for PR #5081. I have addressed the issue of statuses permission there.